intl/en/sl-deploy.txt

```diff
@@ -0,0 +1,64 @@
+usage: {{%MAIN% [options] [URL [PACK|BRANCH]]}}
+
+Deploy a node application to a StrongLoop process manager
+
+Options:
+  {{-h,--help}}          Print this message and exit.
+  {{-v,--version}}       Print version and exit.
+  {{-s,--service SVC}}   Deploy to service SVC.
+  {{-z,--size SIZE}}     Set size of cluster to SIZE before deploy.
+
+Arguments:
+  {{SIZE}}: A number, n, or the string '{{cpus}}', meaning run a worker per CPU.
+
+  {{SVC}}: The name or ID of the service on the process manager.
+
+    Defaults to the name of the application specified in the {{package.json}} file.
+
+    If a service with that name is not available on the process manager,
+    a new service will be created and the application will be deployed.
+
+  {{URL}}: The URL of the StrongLoop process manager.
+
+    Defaults to `{{http://localhost:8701}}`. If a URL is provided, both the host
+    and port is optional, they default to `{{localhost}}` and 8701, respectively.
+    If the server requires authentication, the credentials must be part of the
+    URL, see examples.
+
+    URL may also use the `{{http+ssh://}}` protocol which will connect using ssh
+    and then tunnel the http requests over that connection. The ssh username
+    will default to your current user and authentication defaults to using your
+    current ssh-agent. The {{username}} can be overridden by setting an `SSH_USER`
+    environment variable. The authentication can be overridden to use an
+    existing private key instead of an agent by setting the `SSH_KEY`
+    environment variable to the path of the private key to be used. The port
+    used for ssh can be overridden by setting an `SSH_PORT` environment
+    variable.
+
+  {{PACK}}: Deploy an NPM {{package/tarball}}.
```

Comment:
You don't have to spend time on comments like the above, if you make the change, and push the code, that's all you need. I don't mind, but its time consuming for you (and I won't even see it usually, once you push the branch the comments on the line dissappear if the line is changed).